CHANGELOG.md

```diff
@@ -1,14 +1,49 @@
 # Changelog
 All notable changes to this project will be documented in this file. This CHANGELOG roughly follows the guidelines from [www.keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [v0.11.0]
+### Added
+- Add test for double square brackets to katex-spec (#1956)
+- Add option to render only MathML so that its visible (#1966)
+- Support {smallmatrix}, {subarray}, and \substack (#1969)
+- Enable minRuleThickness in rendering options (#1964)
+- Add \plim (#1952)
+- Support Unicode \digamma (#2010)
+- [breaking] trust setting to indicate whether input text is trusted (#1794)
+- Support \operatorname* (#1899)
+- Re-add includegraphics (#2053)
```

Comment:
This bullet could use rewording, because in the changelog we've never seen an add of includegraphics.  Maybe "Support" or "Support (re-add)"?

```suggestion
- Support \includegraphics, with appropriate trust setting (#2053)
```